packages/frontend/src/index.css

```
@@ -39,20 +39,19 @@
-  --color-homepage-cta-text: var(--homepage-cta-text);
+  --color-wb-bg: var(--wb-bg);
+  --color-wb-paper: var(--wb-paper);
+  --color-wb-ink: var(--wb-ink);
+  --color-wb-sub: var(--wb-sub);
+  --color-wb-rule: var(--wb-rule);
+  --color-wb-syrup: var(--wb-syrup);
+  --color-wb-syrup-deep: var(--wb-syrup-deep);
+  --color-wb-butter: var(--wb-butter);
+  --color-wb-berry: var(--wb-berry);
+  --color-wb-leaf: var(--wb-leaf);
+  --font-display: "Fraunces", ui-serif, Georgia, serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)